### PR TITLE
Fix titlescreen runtime

### LINKED ISSUE #8293: Titlescreens aren't always loaded in
## Round ID & Round Date:

42151 @ 2023-01-07

## Rev info

Round ID: 42141
BYOND Version: 514.1589
Server revision compiled on: 2023-01-06T10:29:22+00:00
Master commit: [011fa789909bcb3d704325ae3f9827e0b4470d06](https://www.github.com/beestation/beestation-hornet/commit/011fa789909bcb3d704325ae3f9827e0b4470d06)
The following pull requests are currently test merged:
[#8264: 'Removes blocks_air' by PowerfulBacon at commit 5f623b4e60](https://www.github.com/beestation/beestation-hornet/pull/8264)
[#8245: 'Pre-generates ruins before explorers arrive' by PowerfulBacon at commit 1b177ec268](https://www.github.com/beestation/beestation-hornet/pull/8245)

Local commit: 0d75fe4e3b239ae75d138016b5853423251940d6
TGS version: 5.2.4
DMAPI version: 5.3.0

## Runtime:

```
[01:51:57] Runtime in title.dm, line 49: Cannot execute null.resetToDefault().
proc name: Initialize (/datum/controller/subsystem/title/Initialize)
src: Title Screen (/datum/controller/subsystem/title)
call stack:
Title Screen (/datum/controller/subsystem/title): Initialize(67175)
Master (/datum/controller/master): Initialize(10, 0, 1)
```

### PR DESCRIPTION
**Pull request by ike709**
## About The Pull Request
Fixes #8293

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog
:cl:
fix: Fixed a runtime that broke titlescreens.
/:cl:

### CODE REVIEW

File: code/controllers/subsystem/title.dm [hunk 1]
```diff
@@ -46,7 +46,10 @@ SUBSYSTEM_DEF(title)
 
 	//Update fast joiners
 	for (var/mob/dead/new_player/fast_joiner in GLOB.new_player_list)
-		fast_joiner.client?.view_size.resetToDefault(getScreenSize(fast_joiner))
+		if(isnull(fast_joiner.client?.view_size))
+			fast_joiner.client?.change_view(getScreenSize(fast_joiner))
+		else
+			fast_joiner.client?.view_size.resetToDefault(getScreenSize(fast_joiner))
```
Comment (Archanial) [inline code/controllers/subsystem/title.dm:52]: you are checking for client 3 times here

**Review by Archanial — DISMISSED** [review]

File: code/controllers/subsystem/title.dm (hunk 1 above)
Comment (ike709) [reply code/controllers/subsystem/title.dm:52]: never trust client to not be null

Comment (Archanial) [reply code/controllers/subsystem/title.dm:52]: It won't change between 2 lines

Comment (ike709) [reply code/controllers/subsystem/title.dm:52]: never trust client to not be null

**Comment by PowerfulBacon** [comment]
I hate client bullshit, I bet this can still runtime

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**